src/Gui/DlgSettingsNavigation.ui

```diff
@@ -7,7 +7,7 @@
     <x>0</x>
     <y>0</y>
     <width>500</width>
-    <height>391</height>
+    <height>472</height>
```

Comment:
80 pixels more for one additional setting seems strange

Reply:
Qt Designer modified this.
That's the first time I use it, so maybe I touched something I shouldn't have.
Should I revert this directly by editing the .ui file ?